chat.php

```diff
@@ -49,15 +49,53 @@ function chat() {
 			$title = $player->rank_name;
 			$staff_level = $player->staff_level;
 			$supported_colors = array(
-				'blue' => -1,
+				'black' => 0,'blue' => -1,
 				'pink' => -2,
 				'gold' => -3,
 				'green' => -4,
 				'teal' => -5,
 				'purple' => -6,
 				'red' => -7,
 			);
```

Comment:
While we're here, can we get rid of this string => number mapping and just store the color names in the DB?